apps/rmh-web/app/pt-portal/page.tsx

```diff
@@ -1,12 +1,14 @@
-'use client'
-
-import '@aws-amplify/ui-react/styles.css';
-import '../globals.css';
-import Head from 'next/head';
-import TopBar from '@/components/TopBar';
+"use client";
 
+import "@aws-amplify/ui-react/styles.css";
+import "../globals.css";
+import Head from "next/head";
+import TopBar from "@/components/TopBar";
+import SideBar from "@/components/SideBar";
+import { useUsers } from "./hooks";
 
 export default function PTPortal() {
+  const { isLoading, users } = useUsers("6b87d552-a2fe-465a-998c-1b288fee212f");
```

Comment:
I think ideally this entire hook is just a redux store to make passing around data a bit easier.  Especially if we start selecting an active user to have their appointment info show in the middle of the portal here